.github/actions/build-binary/action.yml

```diff
@@ -19,22 +19,24 @@ runs:
       with:
         node-version: 16
         cache: "yarn"
-        cache-dependency-path: quickwit-ui/yarn.lock
+        cache-dependency-path: quickwit/quickwit-ui/yarn.lock
     - run: make build-ui
       shell: bash
     - uses: actions-rs/toolchain@v1
       with:
         toolchain: stable
         target: ${{ inputs.target }}
         override: true
-    - uses: actions-rs/cargo@v1
-      with:
-        use-cross: true
```

Comment:
This doesn't work?

Reply:
Unfortunately, GitHub actions do not let you change directories with `working-directory`, and cargo action do not expose any args to specify the rust project directory.